btr-web/btr-common-components/pnpm-lock.yaml

```diff
@@ -1,432 +1,697 @@
-lockfileVersion: '9.0'
+lockfileVersion: '6.0'
 
 settings:
   autoInstallPeers: true
   excludeLinksFromLockfile: false
 
-importers:
-
-  .:
-    dependencies:
-      '@vuepic/vue-datepicker':
-        specifier: ^8.0.0
-        version: 8.2.0(vue@3.4.21)
-      country-codes-list:
-        specifier: ^1.6.11
-        version: 1.6.11
-      keycloak-js:
-        specifier: ^22.0.5
-        version: 22.0.5
-      maska:
-        specifier: 3.0.0-beta2
-        version: 3.0.0-beta2
-      uuid:
-        specifier: ^9.0.1
-        version: 9.0.1
-      vue-country-flag-next:
-        specifier: ^2.3.2
-        version: 2.3.2(vue@3.4.21)
-    devDependencies:
-      '@nuxtjs/i18n':
-        specifier: ^8.2.0
-        version: 8.2.0(rollup@4.13.0)(vue@3.4.21)
-      '@testing-library/vue':
-        specifier: ^8.0.2
-        version: 8.0.2(@vue/compiler-sfc@3.4.21)(vue@3.4.21)
-      '@vitest/coverage-v8':
-        specifier: ^1.3.1
-        version: 1.3.1(vitest@1.3.1)
-      vite:
-        specifier: 5.2.0-beta.1
-        version: 5.2.0-beta.1
-      vitest:
-        specifier: 1.3.1
-        version: 1.3.1
-      vitest-environment-nuxt:
-        specifier: ^1.0.0
-        version: 1.0.0(@testing-library/vue@8.0.2(@vue/compiler-sfc@3.4.21)(vue@3.4.21))(@vue/test-utils@2.4.5)(h3@1.11.1)(rollup@4.13.0)(vite@5.2.0-beta.1)(vitest@1.3.1)(vue-router@4.3.0(vue@3.4.21))(vue@3.4.21)
+dependencies:
```

Comment:
can you revert all the changes in here and just add the new dependency thats needed please? It's very hard to tell what is being updated in here.

We do dependency upgrades outside of regular tickets because it can introduce a lot of minor issues that are hard to find in review

Reply:
Done it's fully reverted because the only thing that happened was a small dependency bump which was unneeded